colour/models/rgb/ycbcr.py

```diff
@@ -0,0 +1,359 @@
+    ...              K=WEIGHT['Rec.601'],
+    ...              inRange=[0, 255],
+    ...              outRange=[0, 255, 0, 256])  # doctest: +ELLIPSIS
+    array([ 36, 136, 175])
+
+    Note the use of 256 for the max Cb/Cr value, which is required so that the
+    Cb and Cr output is centred about 128. Using 255 would centre them about
+    127.5 meaning that there was no integer code value to represent neutral
+    colours. This does however create the possibility of output values of 256,
+    which cannot be stored in an 8-bit integer.
+    """
+    Kr = K[0]
+    Kb = K[1]
+    intOut = (isinstance(outRange[0], int) and isinstance(outRange[1], int) and
+              isinstance(outRange[2], int) and isinstance(outRange[3], int))
+    rgb_float = (rgb.astype(float) - inRange[0])
+    rgb_float *= 1.0/(inRange[1] - inRange[0])
+    r, g, b = tsplit(rgb_float)
+    Y = Kr*r + (1.0 - Kr - Kb)*g + Kb*b
+    Cb = 0.5*(b - Y)/(1.0 - Kb)
+    Cr = 0.5*(r - Y)/(1.0 - Kr)
+    Y *= outRange[1] - outRange[0]
+    Y += outRange[0]
+    Cb *= outRange[3] - outRange[2]
+    Cr *= outRange[3] - outRange[2]
+    Cb += (outRange[3] + outRange[2])/2
+    Cr += (outRange[3] + outRange[2])/2
+    if intOut:
+        Y = np.round(Y).astype(int)
+        Cb = np.round(Cb).astype(int)
+        Cr = np.round(Cr).astype(int)
+    YCbCr = tstack((Y, Cb, Cr))
+    return YCbCr
+
+
+def YCbCr_to_rgb(YCbCr, K=WEIGHT['Rec.709'], inRange=RANGE['legal_10_YC'],
+                 outRange=RANGE['full']):
+    """
+    Converts an array of Y'CbCr values to the corresponding R'G'B' values.
+
+    Note: For ITU-R BT.2020 (Rec.2020) the YCbCr_to_rgb function is only
+    applicable to the non-constant luminance implementation.
+    The YcCbcCrc_to_rgb function should be used for the constant luminance case
+    See https://www.itu.int/dms_pubrec/itu-r/rec/bt/\
+    R-REC-BT.2020-0-201208-S!!PDF-E.pdf
+
+    Parameters
+    ----------
+    YCbCr : array_like
+        Input Y'CbCr values. These may be floats or integers.
+    K : array of two values (optional)
+        The luma weighting coefficients of red and blue.
+        Default: WEIGHT['Rec.709']
+    inRange : array of four values (optional)
+        The values of Y' corresponding to 0 IRE and 100 IRE,
+        and the minimum and maximum values of Cb and Cr.
+        These may be floats or integer code values.
+        Default: RANGE['legal_10_YC']
+    outRange : array of two values (optional)
+        The values of R', G' and B' corresponding to 0 IRE and 100 IRE.
+        These may be floats or integer code values.
+        Default: RANGE['full']
+
+    Returns
+    -------
+    ndarray
+        R'G'B' array.
+        These may be floats or integer code values, depending on the parameter
+        values given for outRange.
+
+    Examples
+    --------
+    >>> YCbCr = np.array([502, 512, 512])
+    >>> YCbCr_to_rgb(YCbCr,
+    ...              inRange=RANGE['legal_10_YC_int'])  # doctest: +ELLIPSIS
```

Comment:
1. Not sure we need the `# doctest: +ELLIPSIS` here.
2. Extreme nitpicking: We usually put the ellipsis statement after the function call opening, and put the arguments on the next lines: https://github.com/colour-science/colour/blob/develop/colour/colorimetry/spectrum.py#L895